docs/getting-started/example-deployment/running-the-deployment.md

```diff
@@ -0,0 +1,235 @@
+# Running the Deployment
+
+To run the containers there are a couple of options but primarily the two main ways are with either
+Docker Compose or Kubernetes (via Helm), the [gaffer-docker](https://github.com/gchq/gaffer-docker)
+repository has some examples of how to run both.
+
+## Docker Compose
+
+For this example we will use a slightly modified version of the docker compose config file used in
```

Comment:
How modified? If it's just volumes, I think it would be easier to use the original file with some additional bits added to the command instead of requiring a whole new file.

Reply:
The existing file has all the build info and file paths/vars for that as well which are irrelevant for the example as we just want to run the existing one, also means don't have to leave the docs to go get another file.

Reply:
Ok, it does makes sense to remove the build sections. In that case there's some comments that should also be removed.

Reply:
Agreed I'll do some tidying of the comments too